demos/browser/app/meetingV2/meetingV2.ts

```diff
@@ -95,7 +95,7 @@ declare global {
 
 class DemoTileOrganizer {
   // this is index instead of length
-  static MAX_TILES = 17;
+  static MAX_TILES = 27;
```

Comment:
Can you please help me understand this bump?
Old: 17 = 16 local video tiles + 1 content share tile.
New: 27 = 25 video tiles + 2 content share tiles? 
I think, this demo does not showcase simultaneous 2 content share tiles.

Reply:
Yes, that is correct.

Reply:
No our demo supports 2 content share if you enable a flag in the URL https://github.com/aws/amazon-chime-sdk-js/blob/0d538ef5b9a05442867557d6c646a4cc5ccae599/demos/browser/app/meetingV2/meetingV2.ts#L2945

Reply:
Ohh, dint knew we had that I think discussed this in standup sometime. Anyways, thanks for correcting. So, then we need also check that support as well as MAX_TILES actually limit. I am not sure how the current `videoTileDidUpdate` binds the second video tile then.

Reply:
There would be no different in term of binding. When we allow 2 content share, the third person would not be able to start another content share.